docker-compose.yml

```diff
@@ -50,6 +50,7 @@ x-shared-unittest-definitions: &shared-unittest-definitions
     args:
       - CLANG_VER=${CLANG_VER:-365097-f7e52fbd-8}
       - COBALT_CLANG_VER=${COBALT_CLANG_VER:-17-init-8029-g27f27d15-3}
+      - CLANG_16_VER=${CLANG_16_VER:-16-init-17653-g39da55e8-2}
```

Comment:
I would actually factor this out from here and hardcode the version in Dockerfiles. Then the hack becomes more localized

Reply:
done.